### PR TITLE
bpo-39366: Remove xpath() and xgtitle() methods of NNTP.

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue39366

### CODE REVIEW

File: Doc/whatsnew/3.9.rst
Comment (vstinner) [inline Doc/whatsnew/3.9.rst]: Please elaborate on the rationale for the removal.

xpath:

```
Most of the time, this extension is not
enabled by NNTP server administrators.
```

xgtitle:

```
This is an optional NNTP extension, and may not be supported by all servers.
:rfc:`2980` says "It is suggested that this extension be deprecated".  Use
:meth:`descriptions` or :meth:`description` instead.
```

File: Doc/whatsnew/3.9.rst
Comment (vstinner) [inline Doc/whatsnew/3.9.rst:354]: Please add an empty line. I'm not sure that it's rendered properly without this newline.

**Review by vstinner — APPROVED** [review]
LGTM. But I would prefer to have a second review, since I'm not used to nntplib.

@serhiy-storchaka @vadmium @berkerpeksag @pitrou: Can one of you please review this PR as well? Well, IMO the PR itself is fine, the question is more if you are ok to drop these two deprecated methods.

File: Doc/whatsnew/3.9.rst
Comment (corona10) [reply Doc/whatsnew/3.9.rst:354]: Updated!

File: Doc/whatsnew/3.9.rst
Comment (berkerpeksag) [inline Doc/whatsnew/3.9.rst]: administrator -> administrator**s**

File: Doc/whatsnew/3.9.rst
Comment (berkerpeksag) [inline Doc/whatsnew/3.9.rst]: Maybe add something like "generally" or "usually" to make it less strong?

File: Misc/NEWS.d/next/Library/2020-01-17-18-14-51.bpo-39366.Cv3NQS.rst
Comment (berkerpeksag) [inline Misc/NEWS.d/next/Library/2020-01-17-18-14-51.bpo-39366.Cv3NQS.rst]: Personally I'd prefer something like this:

```rst
The previously deprecated ``xpath()`` and ``xgtitle()`` methods of :class:`nntplib.NNTP` have been removed.
```

File: Lib/nntplib.py
Comment (berkerpeksag) [inline Lib/nntplib.py:843]: Please drop the unused `warnings` import in line 70.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

**Review by corona10 — COMMENTED** [review]
@berkerpeksag 

Thanks for the review!
I have made the requested changes; please review again

**Review by vstinner — APPROVED** [review]
LGTM. @berkerpeksag: @corona10 addressed your remarks, would you mind to review it again?

**Review by berkerpeksag — APPROVED** [review]
Looks great, thank you!

**Pull request merged**
